reference/test/matrix/dense_kernels.cpp

```diff
@@ -2009,6 +2009,74 @@ TYPED_TEST(Dense, ExtractsDiagonalFromNonSquareMatrix)
 }
 
 
+TYPED_TEST(Dense, InplaceAbsolute)
+{
+    using T = typename TestFixture::value_type;
+    // clang-format off
+    // {1.0, -1.0, -0.5},
+    // {-2.0, 2.0, 4.5},
+    // {2.1, 3.4, 1.2}
+    // clang-format on
+    this->mtx5->compute_absolute_inplace();
```

Comment:
nit: spacing for AAA (in all added tests in this file):
```suggestion
    // clang-format on
    
    this->mtx5->compute_absolute_inplace();
```